src/data_gen.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
It won't work on Windows. Is it possible to use something different?

Reply:
Like what ? I could write a caml executable and call it instead of having a shell script, what do you think ?

Reply:
I suppose an OCaml executable will do. Can you use an OCaml script instead:
ocaml src/data_gen.ml

Reply:
Let me know if you want to fix this or if I should fix it myself. I am working on Windows right now, so it might be easier for me to find a solution and test it.

Reply:
I fixed it yesterday. I used an executable and not an ocaml script, because it's easier when you want to debug (you can run the executable yourself).